app/build.gradle

```diff
@@ -15,6 +15,7 @@ android {
         versionName "3.0"
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        resValue("string", "app_version", "${versionName} (build $versionCode)")
```

Comment:
No need to do this. You can use `BuildConfig` for that. Please remove this.

Reply:
I needed it in string res folder you can see it in string folder , Since there is no other way to pass value in preferences which is custom build., you can't pass values to XML file using Build.Config, Trust me wasted 2-3hrs on this.

Reply:
Why you needed that in `string.xml`?

Reply:
Yep, Sorry it was not in string res folder kind of slip out in hurry.